examples/extract-table-from-image/index.ts

```diff
@@ -0,0 +1,70 @@
+import Instructor from "@/instructor"
+import OpenAI from "openai"
+import { z } from "zod"
+
+const IMAGE_URL = "https://duquark.files.wordpress.com/2019/12/graph.jpg?w=700&h=499"
+
+const TableResponseSchema = z.object({
+  title: z.string(),
+  caption: z.string(),
+  table: z.any()
+})
+type TableResponse = z.infer<typeof TableResponseSchema>
+
+const oai = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY,
+  organization: process.env.OPENAI_ORG_ID
+})
+
+const instructor = Instructor({
+  client: oai,
+  mode: "MD_JSON"
+})
+
+const extractTableFromImage = async (imageUrl: string): Promise<TableResponse | undefined> => {
+  try {
+    const response = await instructor.chat.completions.create({
+      model: "gpt-4-vision-preview",
+      response_model: {
+        schema: TableResponseSchema,
+        name: "table extraction"
+      },
+      max_tokens: 1_000,
```

Comment:
typo?

Reply:
that's intended :), check out https://www.javascripttutorial.net/es-next/javascript-numeric-separator/

Reply:
oh cool - I've never used this before!